src/test/java/com/cloudbees/jenkins/support/config/SupportAutomatedBundleConfigurationSystemPropertiesTest.java

```diff
@@ -0,0 +1,110 @@
+package com.cloudbees.jenkins.support.config;
+
+import com.cloudbees.jenkins.support.SupportPlugin;
+import com.cloudbees.jenkins.support.api.Component;
+import org.htmlunit.html.HtmlElement;
+import org.htmlunit.html.HtmlForm;
+import org.htmlunit.html.HtmlInput;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RealJenkinsRule;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+
+/**
+ * Test for the {@link SupportAutomatedBundleConfiguration}
+ * @author Allan Burdajewicz
+ */
+public class SupportAutomatedBundleConfigurationSystemPropertiesTest {
+
+    @Rule
+    public RealJenkinsRule rr = new RealJenkinsRule();
+
+    @Test
+    public void enforcePeriod() throws Throwable {
+        rr.javaOptions().javaOptions("-D" + SupportPlugin.class.getName()
+            + ".AUTO_BUNDLE_PERIOD_HOURS=2").then(r -> {
```

Comment:
May not be safe to pass lambdas to `then`. (Serialized into a different JVM.) Generally suggest using a `static` nested class when parameters must be passed, and otherwise (normal case) using a `static` method reference.

Reply:
That explains why I am not able to debug inside the lambda..